src/Files.App/App.xaml.cs

```diff
@@ -123,6 +124,7 @@ private IServiceProvider ConfigureServices()
 				.AddSingleton<IThreadingService, ThreadingService>()
 				.AddSingleton<ILocalizationService, LocalizationService>()
 				.AddSingleton<ICloudDetector, CloudDetector>()
+				.AddSingleton<IKeyboardManager, KeyboardManager>()
```

Comment:
```suggestion
				.AddSingleton<IKeyboardAcceleratorrManager, KeyboardManager>()
```

Reply:
I mentioned in the description that it was in progress. It will indeed be cleaner with 2 managers. I mentioned in the description that it was in progress. It will indeed be cleaner with 2 managers. I will do this this weekend at the latest.